test/e2e-api/admin/members.test.js

```diff
@@ -1554,7 +1554,7 @@ describe('Members API', function () {
                 'content-disposition': anyString
             });
 
-        res.text.should.match(/id,email,name,note,subscribed_to_emails,complimentary_plan,stripe_customer_id,created_at,deleted_at/);
```

Comment:
Is there a reason to keep these 2 tests in `members.test.js` when we've got a suite in `members-exporter.test.js` now? Seems like they are duplicates?

Reply:
I don't have any tests yet for the search option in the other file, so I think it is best to keep this one until we have a new one for search

Reply:
One of them is no search,  I just noticed. We can clean it up later :)